jib-core/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- To disable parallel execution, the property `jib.serialize` should be used instead of `jibSerialize`. ([#1968](https://github.com/GoogleContainerTools/jib/issues/1968))
```

Comment:
FYI I think we can leave the changelog updates in the original PR, since those are explicit about what's still unreleased, just not the documentation.

Reply:
Haha, yeah you're right.